modules/vault-secrets-darwin.nix

```diff
@@ -0,0 +1,80 @@
+# SPDX-FileCopyrightText: 2020 Serokell <https://serokell.io/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.vault-secrets;
+in
+{
+  options = import ./options.nix { inherit lib cfg; };
+
+  config = {
+    #systemd.services = lib.mkMerge ([(flip mapAttrs' cfg.secrets (
+    launchd.daemons = lib.mkMerge [(flip mapAttrs' cfg.secrets (
+      name: scfg: with scfg;
+      let
+        secretsPath = "${cfg.outPrefix}/${name}";
+      in nameValuePair "${name}-secrets" {
+        path = with pkgs; [ getent jq vault-bin python3 ];
+
+        environment.VAULT_ADDR = cfg.vaultAddress;
+
+        script = ''
```

Comment:
AFAICS, this script duplicates the one from `modules/vault-secrets.nix`, I think it's worth extracting it to avoid duplication